erpnext/accounts/doctype/sales_invoice/sales_invoice.js

```diff
@@ -447,6 +447,15 @@ erpnext.accounts.SalesInvoiceController = erpnext.selling.SellingController.exte
 		this.frm.refresh_field("outstanding_amount");
 		this.frm.refresh_field("paid_amount");
 		this.frm.refresh_field("base_paid_amount");
+	},
+
+	conversion_rate() {
+		this._super();
+		$.each(cur_frm.doc.timesheets, function(i, d) {
+			let row = frappe.get_doc(d.doctype, d.name)
+			frappe.model.set_value(d.doctype, d.name, 'billing_amount', flt(d.billing_amount)*flt(cur_frm.doc.conversion_rate))
+		});
+		calculate_total_billing_amount(cur_frm)
```

Comment:
On changing currency, the amount item table and timesheets are different. Am I missing something?


https://user-images.githubusercontent.com/24353136/123066130-6818ba80-d42d-11eb-8584-4f92d65f241a.mp4